docs/_feb-milestone_.md

```diff
@@ -0,0 +1,56 @@
+## Monthly Milestone - Feb 2020
+
+### Overview
+
+- Continue focus on migration guide
+
+### Goals
+
+- Migration guide between LB3 and LB4 MVP #453
+
+  - [ ] (3) How to map LB3 CLI commands to LB4 CLI #3953
+  - [ ] (3) How to migrate CRUD Operation Hooks #3952
+  - [ ] (8) How to migrate model mixins #3951
+  - [ ] (3) Spike: how to import LB3 models extending a custom base model #3813
+
+- Tooling around migration
+
+  - [ ] (3) LB3 model import: preserve connector-specific metadata in property
+        definitions #3810
+
+- Create models & REST APIs dynamically at runtime
+
+  [ ](8) How to build models, repositories and controllers dynamically at
+  runtime #4296 [needs estimate]
+
+- Shopping app web site
+
+  - [ ] (2) Shopping App Website: Installation/setup documentation
+        https://github.com/strongloop/loopback4-example-shopping/issues/477
+
+- (3) Run shared tests from both v3 and v4 of juggler
+  https://github.com/strongloop/loopback-connector-db2/issues/133
+
+- (3) Add CrudRestApiBuilder to `@loopback/rest-crud` #3737
+
+- PR reviews:
+  - [ ] feat: adds @deprecated convenience decorator #4415
+  - [ ] feat: adds @tags convenience decorator #4416
+  - [ ] feat: adds utility launch for debugging a single test file #4432
+  - [ ] Add title to filter schemas #4355
+
+### Stretch Goals
+
+- Migration guide between LB3 and LB4 MVP #453
+
+  - [ ] Which official LB3 components are not planned in LB4 #3956
+
+- Developer Experience
+
+  - [ ] Sugar decorators for Filter and Where parameters #1749
+
+- CI
```

Comment:
Can we please prioritize CI fixes higher? Either make them the first stretch goals to look at, or even move them to regular (committed) tasks.